Sound/Tidal/Pattern.hs

```diff
@@ -1116,6 +1116,44 @@ within (s,e) f p = stack [playWhen (\t -> cyclePos t >= s && cyclePos t < e) $ f
                           playWhen (\t -> not $ cyclePos t >= s && cyclePos t < e) $ p
                          ]
 
+{- |
+For many cases, @within'@ will function exactly as within.  
+The difference between the two occurs when applying functions like 'fast' or '<~'. 
+within applies the function to the whole cycle first then keeps notes only in the specified interval. 
+within', however, only sees the notes in the specified interval to which it then applies the function.  
```

Comment:
I'd write:

within' keeps notes only in the specified interval first, and then applies the function.

This makes the symmetry clear.